features/input_validation.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Sign up Input Validation
+  As a user that inserts data into platform
+  In order to know how to fill in the right data
+  I want to have fill in support messages
+
+  Scenario: First Name can't be blank
+    Given a new user a with a blank first name field
```

Comment:
Maybe make this test more form specific.

You could just create a background like:
```
Background:
  Given A new user
  When he visits the new user page
  And he enters his information
```

and a specific scenario like:

```
  When he enters an empty first name
  And submits
  Then he should not be able to sign up
```